modules/@apostrophecms/attachment/index.js

```diff
@@ -285,9 +285,18 @@ module.exports = {
           return;
         }
         _.assign(info, _.omit(dbInfo, 'crop'));
-        if (!self.acceptableExtension(field, info)) {
-          throw self.apos.error('invalid');
+
+        // Check if the file type is acceptable of if there are
+        const correctedExtensions = self.checkExtension(field, info);
```

Comment:
I wanted to get the list of acceptable extensions from this, so I changed how this works.